tests/ui/attributes/unix_sigpipe/auxiliary/sigpipe-utils.rs

```diff
@@ -17,17 +17,17 @@ pub fn assert_sigpipe_handler(expected_handler: SignalHandler) {
         target_os = "android",
     )))]
     {
-        let prev = unsafe { libc::signal(libc::SIGPIPE, libc::SIG_IGN) };
+        let actual = unsafe {
+            let mut actual: libc::sigaction = std::mem::zeroed();
```

Comment:
sad that libc doesn't have a good way to init this, can't even do it fully safely manually, because some fields have private fields :(